### PR TITLE
sort users by first_name and last_name

### PR DESCRIPTION
**Pull request by oleg08**
code review:
- [x] @DimaSmagulov
- [ ] @CHILL99 
- [ ] @krisberry 
- [x] @natalya5 
- [x] @OrestF 
- [x] @rtershak 
- [x] @vvs79 

2nd lvl review:

- [x] @Kmalynss
- [ ] @serhiym 
- [ ] @ruba-ruba

https://trello.com/c/ycF8Rxzu/42-as-an-admin-when-on-users-index-page-i-want-to-be-able-to-sort-users-by-first-name-last-name-etc

### CODE REVIEW

**Comment by OrestF** [comment]
I its not a good idea to add pictures like this app/assets/images/arrow-3D-green-down.jp
Try to use glyphicons (bootstrap icons).

File: app/assets/stylesheets/custom.css.scss [hunk 1]
```diff
@@ -244,3 +244,17 @@ td {
...
+
+.pretty .current {
+  padding-right: 12px;
+  background-repeat: no-repeat;
+  background-position: right center;
+}
+
+.pretty .asc {
+  background-image: url(arrow-3D-green-up.jpg);
+}
```
Comment (Kmalynss) [inline app/assets/stylesheets/custom.css.scss]: use bootstrap [glyphicons](http://getbootstrap.com/components/#glyphicons)

File: app/controllers/admin/users_controller.rb [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
+  helper_method :sort_column, :sort_direction
 
   def index
     if params[:waiting_approval]
       @users = User.waiting_approval
     else
-      @users = User.all
+      @users = User.order("#{sort_column} #{sort_direction}")
     end
   end
```
Comment (Kmalynss) [inline app/controllers/admin/users_controller.rb]: We should be able to sort users in both cases

Comment (Kmalynss) [inline app/controllers/admin/users_controller.rb]: move ```@user=``` out of if..esle.. statement like:
```ruby
@users = if ...
  ...
else
  ...
end

**Comment by vvs79** [comment]
reviewed

**Comment by DimaSmagulov** [comment]
reviewed

File: app/helpers/application_helper.rb [hunk 3]
```diff
@@ -2,4 +2,4 @@ module ApplicationHelper
   def full_title(page_title = nil)
     [page_title, "AID-Interview"].compact.join(" | ")
   end
-end 
+end
```
Comment (Kmalynss) [inline app/helpers/application_helper.rb]: don't forget new line symbol

File: app/views/admin/users/index.html.slim [hunk 4]
```diff
@@ -1,18 +1,19 @@
...
       = link_to "All Users", admin_users_path, class: 'btn btn-default role="button"'
       = link_to "Users awaiting approval", admin_users_path(waiting_approval: true), class: 'btn btn-default role="button"'
-      = link_to "Questions", admin_questions_path, class: 'btn btn-default role="button"'          
+      = link_to "Questions", admin_questions_path, class: 'btn btn-default role="button"'
       = link_to "Topics", admin_topics_path, class: 'btn btn-default role="button"'
   h2 Interviewers
   table.table.table-bordered.table-hover
     tr.active
-      th width="30%"
+      th style="width:30%;" = sort_link "first_name"
```
Comment (serhiym) [inline app/views/admin/users/index.html.slim]: move sort_link to new line
```
th style
 = sort_link
 h4 First Name
```

**Comment by natalya5** [comment]
reviewed

**Pull request merged**